core/src/main/resources/hudson/PluginManager/check.jelly

```diff
@@ -25,7 +25,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
 
   ${%lastUpdated(app.updateCenter.lastUpdatedString)}
-  <f:link href="checkUpdatesServer" post="true" clazz="yui-button yui-submit-button submit-button primary">
+  <f:link href="checkUpdatesServer" post="true" clazz="jenkins-button jenkins-button--primary submit-button">
```

Comment:
I think we should have only one primary button on a page.:

```suggestion
  <f:link href="checkUpdatesServer" post="true" clazz="jenkins-button submit-button">
```

Reply:
> only one primary button on a page

Is there are a particular reason, why the blue button should be turned into a less intuitive and harder to distinguish, default button?

![Screenshot 2022-09-29 at 23 51 38](https://user-images.githubusercontent.com/13383509/193148636-3b49ef33-9b35-421f-a39b-4729e8ce2a09.png)

Reply:
If you think that default button has a bad design then we need to improve that in a different PR. My comment is only about the semantics of your button choice. 

An important UI design rule is that a page has a single primary action only (example: https://uxplanet.org/primary-secondary-action-buttons-c16df9b36150). On the updates and install pages this should be the "Install" action. The check for updates button certainly is only a secondary action, hence I would use the associated class for this action.

Reply:
> Which one is primary?

Good point. I was going off the default view and had only a couple of updates available, but I do understand the problem with having several primary buttons, that not really are meant to be primary.

Thanks for the feedback & clarification! :)